app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -90,9 +91,20 @@ def refresh_targets_for_ems(ems, targets)
 
           Benchmark.realtime_block(:save_inventory) { save_inventory(ems, target, hashes) }
           _log.info "#{log_header} Refreshing target #{target.class} [#{target.name}] id [#{target.id}]...Complete"
+
+          if hashes.kind_of?(Array)
```

Comment:
right, but the problem is that we throw away the Persister too now. All we send is a list of ICs. Might be good to refactor it all to pass Persister though.